packages/jest-cli/src/cli/index.js

```diff
@@ -101,6 +102,33 @@ export const runCLI = async (
     );
   }
 
+  const {openHandles} = results;
+
+  if (openHandles && openHandles.length) {
+    const handles = openHandles
+      .map(({title, entries}) => {
+        const table = new Table();
+
+        entries.forEach(({file, line}) => {
+          table.cell('File', file);
+          table.cell('Line', line);
+          table.newRow();
+        });
+
+        return title + '\n' + table.toString();
+      })
+      .join('\n\n');
+
+    const message =
+      chalk.red(
+        '\nJest has detected the following ' +
+          `${openHandles.length} open handles potentially keeping Jest from ` +
```

Comment:
Maybe we can use `pluralize`? and rephrase to:
```
Jest has detected 1 open handle potentially preventing it from exiting:
```

Reply:
We have our own `pluralize` in `jest-matcher-utils`. I can move it to `jest-util`, perhaps?

Reply:
There's one in `jest-cli` too. Feel free to clean it up :D

Reply:
ah didn't know about that one. I'll be lazy and just use it then